packages/atomic/cypress/integration/facet-standard.cypress.ts

```diff
@@ -0,0 +1,413 @@
+      expect(analyticsBody).to.have.property(
+        'eventValue',
+        'showLessFacetResults'
+      );
+    });
+  });
+
+  describe('When click ClearAll facet', () => {
+    it('Should clear all checkboxes and log UA', async () => {
+      cy.get('@facetLi_1').click();
+      cy.get('@facetLi_2').click();
+      cy.get('@facetShadow').find(FacetSelectors.clearAllButton).click();
+      cy.get('@facetLi_1')
+        .find(FacetSelectors.checkbox)
+        .should('not.be.checked');
+      cy.get('@facetLi_2')
+        .find(FacetSelectors.checkbox)
+        .should('not.be.checked');
+      const analyticsBody = (await getAnalyticsAt('@coveoAnalytics', 3)).request
+        .body;
+      expect(analyticsBody).to.have.property('actionCause', 'facetClearAll');
+      expect(analyticsBody.facetState).to.have.lengthOf(0);
+    });
+  });
+});
+
+describe('Facet with no facetSearch, and numberOfValues is 5 ', () => {
+  beforeEach(() => {
+    setupPager(
+      facetProp.field,
+      facetProp.label,
+      'enable-facet-search=false number-of-values=5'
+    );
+    createAliasShadow(facetProp.field);
+    createAliasFacetUL(facetProp.field);
+    cy.get('@facet_ul').find('li:nth-child(1)').as('facetLi_1');
+    cy.get('@facet_ul').find('li:nth-child(2)').as('facetLi_2');
+  });
+
+  it('Facet should load without facet search and 5 facet values', async () => {
+    componentLoaded(facetProp.label);
+    cy.get('@facetShadow')
+      .find(FacetSelectors.facetSearchbox)
+      .should('not.exist');
+    validateNumberofValue(5);
+  });
+
+  it('Should active checkbox and log UA', async () => {
+    await validateBasicFacetFunctionality('@facetLi_1', facetProp.field);
+  });
+});
+
+describe('Facet with different sort-criteria options', () => {
+  it('Should using "automatic" sort for default setting', async () => {
+    setupPager(facetProp.field, facetProp.label);
+    await facetSortShouldApply(sortCriteriaOption.automatic, 0);
+  });
+
+  it('Should using "alphanumeric" sort for custom setting', async () => {
+    setupPager(
+      facetProp.field,
+      facetProp.label,
+      'sort-criteria="alphanumeric"'
+    );
+    await facetSortShouldApply(sortCriteriaOption.alphanumeric, 0);
+  });
+
+  it('Should using "occurrences" sort for custom setting', async () => {
+    setupPager(facetProp.field, facetProp.label, 'sort-criteria="occurrences"');
+    await facetSortShouldApply(sortCriteriaOption.occurrences, 0);
+  });
+
+  it('Should using "score" sort for custom setting', async () => {
+    setupPager(facetProp.field, facetProp.label, 'sort-criteria="score"');
+    await facetSortShouldApply(sortCriteriaOption.score, 0);
+  });
+
+  it('Should using "automatic" sort for custom setting', async () => {
+    setupPager(facetProp.field, facetProp.label, 'sort-criteria="automatic"');
+    await facetSortShouldApply(sortCriteriaOption.automatic, 0);
+  });
+
+  describe('Trigger ShowMore on facet with custom sort-criteria', () => {
```

Comment:
Maybe a clearer description would be `... on a facet with sort-criteria other than 'automatic'`